files/build_templates/docker_image_ctl.j2

```diff
@@ -404,28 +404,33 @@ start() {
 
     DOCKERCHECK=`docker inspect --type container ${DOCKERNAME} 2>/dev/null`
     if [ "$?" -eq "0" ]; then
-        {%- if docker_container_name == "database" %}
-        DOCKERMOUNT=""
+        {%- if docker_container_name == "database" and sonic_asic_platform == "vs" %}
```

Comment:
fix indent? Why do we need to make this change? This means every time a vs switch is rebooted, the database switch will be recreated?

Reply:
Indentation seems fine to me? Should it be tab?

yes, this change means database container will be recreated everytime after reboot.. This is to support converting single asic image to multi-asic(based on asic.conf)..That requires database containers to be recreated again.